pkg/constants/env_variables.go

```diff
@@ -5,7 +5,7 @@ package constants
 
 // environment variables for http proxy
 const (
-	ProxyCACert = "PROXY_CA_CERT"
+	ProxyCACert = "TANZU_CLI_PROXY_CA_CERT"
```

Comment:
Yes, it is just to be consistent with naming variables with TANZU_CLI prefix and also I feel it would be better to have a sperate variable for CLI , just incase if TKG uses the environmnet variable for cluster use cases where it may not be applicable to CLI. Having said that, I see your point, that any organization would be having a single proxy and it would be applicable for all the cases. I can change it to `PROXY_CA_CERT` so that it won't cause any backward compatibility issue with existing CLI

Reply:
Yeah. I feel using the old variable name makes sense. Especially considering this will go hand in hand with `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY`, and `PROXY_CA_CERT`.